app/grandchallenge/challenges/models.py

```diff
@@ -102,20 +105,51 @@
 class ChallengeSet(models.QuerySet):
     def with_available_compute(self):
         return self.annotate(
-            approved_compute_costs_euro_millicents=(
+            complimentary_compute_costs=(
```

Comment:
Good to add units for these three new annotations, `complimentary_compute_costs_euros`, so that developers know what they're dealing with.